bindings/http/http.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 // HTTPSource is a binding for an http url endpoint invocation
-// nolint:golint
+//revive:disable
```

Comment:
Comment valid for all changes in this PR.

`//nolint:golint` used to either work for one specific block, or for a single line if specified inline.

`//revive:disable` comments are valid for the entire file. They need to be enabled again with `//revive:enable` after the block ends. For inline usage, we can use `//revive:disable-next-line`.

Reply:
gotcha